lib/versioning/node/index.ts

```diff
@@ -27,8 +29,23 @@ function getNewValue({
 
 export { isValid };
 
+export function isStable(version: string, now = Date.now()): boolean {
+  if (npm.isStable(version)) {
+    const major = npm.getMajor(version);
+    const schedule = nodeSchedule[`v${major}`];
+    if (schedule) {
+      const { maintenance, end } = schedule;
+      const ltsStart = moment(maintenance);
+      const ltsFinish = moment(end);
+      return moment(now).isBetween(ltsStart, ltsFinish);
```

Comment:
It's still stable after the end date. After then it becomes *deprecated* but that's a separate topic not for now.